ui/components/component-library/header-base/header-base.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useRef, useLayoutEffect, useMemo, useState } from 'react';
+import classnames from 'classnames';
+import {
+  BLOCK_SIZES,
+  DISPLAY,
+  JustifyContent,
+} from '../../../helpers/constants/design-system';
+import Box from '../../ui/box';
+
+import { HeaderBaseProps, CustomBoxProps } from './header-base.types';
+
+export const HeaderBase: React.FC<HeaderBaseProps> = ({
+  startAccessory,
+  endAccessory,
+  className,
```

Comment:
Does this not error? I had to define it as a string. Add you're doing a `||` operator should we standardize this? Our options are use the or operator like you're doing or define it as default
```
  endAccessory,
  className = '',
  children,
```
Or
```
className={classnames('mm-header-base', className || '')}
```
I don't think I have a preference, do you?